### PR TITLE
Add tool tip for each button on the landing page form

### PR DESCRIPTION
**Pull request by rileydanejohnston**
- added tool tips for each button
- fix button bug -> buttons loaded to form were submit type buttons
- minor refactoring of the LandingForm states

future changes:
- border shadow for the popup bubble tail. I could not figure it out for the life of me.

- [ ] My code is covered by unit tests 
- [ x] This PR is less than [300 lines of code](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests)
- [ x] My code [smells good](https://en.wikipedia.org/wiki/Code_smell) 
- [ x] I have tested my PR locally and proven it works. 
- [ x] There is [no copy-paste](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

### CODE REVIEW

File: frontend/src/components/landing-form/LandingForm.js [hunk 1]
```diff
@@ -18,33 +18,28 @@ import {
...
-
+  
   const handleBtnClick = (e) => {
     // update isSelected for button that was clicked
     const updateSelectValue = allButtons.map((btn) => {
-      if (e.target.name === btn.name) {
-        return { text: btn.text, name: btn.name, isSelected: !btn.isSelected };
+      if (e.target.name === btn.name)
+      {
+        return { text: btn.text, name: btn.name, isSelected: !btn.isSelected, helpText: btn.helpText };
```
Comment (thinklikeadesigner) [inline frontend/src/components/landing-form/LandingForm.js:42]: This is awesome, I really like how you updated multiple values at once

Comment (McCambley) [reply frontend/src/components/landing-form/LandingForm.js:42]: Agreed, this is super smart, love it.

File: frontend/src/components/landing-help-icon/HelpIcon.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { QuestionMark, PurpleQMark, HelpWrapper, HelpText } from './styledHelpIcon'
+
+export default function HelpIcon({ helpText, custom }) {
+
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  return (
+    custom === false ? 
+    (
+    <QuestionMark onMouseEnter={() => {setIsOpen(true)}} onMouseLeave={() => {setIsOpen(false)}} >
```
Comment (McCambley) [inline frontend/src/components/landing-help-icon/HelpIcon.js:11]: Let's goo, love to see it.

File: frontend/src/components/landing-help-icon/HelpIcon.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { QuestionMark, PurpleQMark, HelpWrapper, HelpText } from './styledHelpIcon'
+
+export default function HelpIcon({ helpText, custom }) {
+
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  return (
+    custom === false ?
```
Comment (McCambley) [inline frontend/src/components/landing-help-icon/HelpIcon.js:9]: Is this conditional a set up for future iterations where investors add their own buttons that might not have help text? Smart idea, thinking ahead like this :)

Comment (McCambley) [reply frontend/src/components/landing-help-icon/HelpIcon.js:9]: Oh wait nevermind, this is used as the purple question mark too, super reusable, got it, also very cool!

File: frontend/src/components/landing-help-icon/styledHelpIcon.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  width: 359px;
+  padding: 24px 20px;
+  border-radius: 20px;
+  position: absolute;
+  top: calc(50% + 28px);
+  left: calc(50% - 20px);
+  z-index: 2;
+  box-shadow: -2px 4px 4px rgba(0, 0, 0, 0.25), inset -3px 2px 2px rgba(255, 255, 255, 0.25), inset 3px -4px 4px rgba(0, 0, 0, 0.25);
+  transition: opacity .3s, visibility .3s;
+  &:before {
```
Comment (McCambley) [inline frontend/src/components/landing-help-icon/styledHelpIcon.js:38]: This part seems tricky. I'm not sure how to get that shadow to appear as it does in the design, but perhaps that can be a future ticket we look at.

**Review by McCambley — APPROVED** [review]
Hey Riley, this all looks really good. Smart implimentations all around. The tooltip arrow ::before bit looks a little weird, but that seems like something that's really hard to translate from the figma over to code. I'll approve this and we can tackle piece later.

**Pull request merged**